src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> STORAGE = new HashMap<>();
```

Comment:
```suggestion
    public static final Map<String, Integer> storage = new HashMap<>();
```
this is not a magic number or something like this
the field can be just static final and not a constant (if it is not a "magic" data)

sorry for the previous mentor comment

Reply:
ok)